sycl/test/lit.cfg.py

```diff
@@ -161,47 +150,52 @@ def getDeviceCount(device_type):
         gpu_check_on_linux_substitute = "| FileCheck %s"
         if cuda:
             gpu_run_on_linux_substitute += " SYCL_BE=PI_CUDA "
+else:
+    lit_config.warning("GPU device not found")
```

Comment:
Not having GPU is a pretty common situation. Should it be a note instead?

Reply:
I would like this to be a warning - it clearly highlights that the coverage of testing is reduced (in terms of devices). Just to avoid the situations when someone wanted to fix something on GPU, but forgot to properly configure environment - all tests passed, but not on GPU